nservicebus/hosting/docker-host/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Docker Container Host
+summary: Take advantage of process isolation by hosting endpoints in Docker containers
+tags:
+ - Hosting
+related:
+ - nservicebus/dotnet-templates
+ - nservicebus/licensing
+ - samples/hosting/docker
+component: Templates
+reviewed: 2018-02-28
+---
+
+Docker containers provide the ability to deploy endpoints in a self-contained manner. To create an endpoint to host in a Docker container, download and install the [Docker endpoint template](/nservicebus/dotnet-templates.md) and create a new project using `dotnet new nsbdockerendpoint`. The project that is created will have the required endpoint setup infrastructure in addition to the needed `DOCKERFILE` to create and deploy a container hosting one endpoint.
```

Comment:
The second sentence should just be "install", not "download and install" since you don't have to download it separately. It installs via NuGet.

Reply:
The nservicebus/dotnet-templates.md page doesn't seem to have been updated to mention anything about the new template.